src/features/operator/Market/MarketOverview/BalanceAndPriceMobile.tsx

```diff
@@ -0,0 +1,84 @@
+import { InfoCircleOutlined } from '@ant-design/icons';
+import { Col, Row, Typography } from 'antd';
+import React from 'react';
+
+import type { MarketInfo } from '../../../../api-spec/protobuf/gen/js/tdex-daemon/v1/types_pb';
+import { CurrencyIcon } from '../../../../common/CurrencyIcon';
+import type { Asset } from '../../../../domain/asset';
+import type { NetworkString } from '../../../../domain/misc';
+import type { LbtcUnit } from '../../../../utils';
+import { formatCompact, isLbtcAssetId, unitToExponent } from '../../../../utils';
+
+const { Title } = Typography;
+
+interface BalanceAndPriceMobileProps {
+  baseAsset: Asset;
+  quoteAsset: Asset;
+  baseAmount?: string;
+  quoteAmount?: string;
+  marketInfo?: MarketInfo;
+  network: NetworkString;
+  lbtcUnit: LbtcUnit;
+}
+
+export const BalanceAndPriceMobile = ({
+  baseAsset,
+  quoteAsset,
+  marketInfo,
+  lbtcUnit,
+  network,
+  baseAmount,
+  quoteAmount,
+}: BalanceAndPriceMobileProps): JSX.Element => {
+  return (
+    <Col span={24}>
+      <div className="panel panel__grey mt-4">
+        <Row>
+          <Col span={24}>
+            <Title className="dm-sans dm-sans__x dm-sans__bold dm-sans__grey d-inline mr-4" level={3}>
+              Market Balance
+            </Title>
+            <InfoCircleOutlined className="grey" />
+          </Col>
+        </Row>
+        <Row align="middle" className="mt-1">
+          <Col className="d-flex align-center">
+            <div className="d-flex align-center">
```

Comment:
I woudl put one asset per row

With long numbers, it can get cut

<img width="356" alt="image" src="https://user-images.githubusercontent.com/3596602/209374443-7eafc29c-6d4c-4cfa-a6ce-22d08011a316.png">